app/views/leagues/show.html.haml

```diff
@@ -1,3 +1,7 @@
+- content_for :league_header_image do
+  - if @league.league_header_image?
```

Comment:
maybe this is for you then: https://github.com/carrierwaveuploader/carrierwave/#providing-a-default-url